### PR TITLE
Prevent CoreWCF to interfer with ASP.NET Core MVC Controller FromServ…

### LINKED ISSUE #635: CoreWCF source generator interfers with usages of [FromServicesAttribute] in ASP.NET Core MVC Controllers
Installing CoreWCF packages in an ASP.NET Core application that already expose routes MVC framework is triggering false positive build errors. The CoreWCF source generator requires the user implemented `Controller` classe to be `partial`.

### PR DESCRIPTION
**Pull request by g7ed6e**
Fix #635

### CODE REVIEW

File: src/CoreWCF.BuildTools/src/AnalyzerReleases.Shipped.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|--------------------
 COREWCF_0100 | OperationParameterInjectionGenerator | Error | COREWCF_0100_OperationParameterInjectionGenerator
 COREWCF_0101 | OperationParameterInjectionGenerator | Error | COREWCF_0101_OperationParameterInjectionGenerator
-COREWCF_0102 | OperationParameterInjectionGenerator | Error | COREWCF_0102_OperationParameterInjectionGenerator
\ No newline at end of file
+COREWCF_0102 | OperationParameterInjectionGenerator | Error | COREWCF_0102_OperationParameterInjectionGenerator
+
+# Release 1.0.0
```
Comment (g7ed6e) [inline src/CoreWCF.BuildTools/src/AnalyzerReleases.Shipped.md:11]: @mconnew please update to what next version number will be

Comment (mconnew) [reply src/CoreWCF.BuildTools/src/AnalyzerReleases.Shipped.md:11]: As long as there's no new api, the next version will be 1.0.1

Comment (g7ed6e) [reply src/CoreWCF.BuildTools/src/AnalyzerReleases.Shipped.md]: the version number is updated

File: src/CoreWCF.BuildTools/tests/OperationParameterInjectionGeneratorTests.cs [hunk 2]
```diff
@@ -1623,9 +1666,134 @@ public partial class IdentityService : IIdentityService
...
+            {
+                TestState =
+                {
+                    Sources =
+                    {
+@$"
+namespace MyProject
+{{
+    //[{attributeNamespace}.ServiceContract]
+    public interface IIdentityService
```
Comment (mconnew) [inline src/CoreWCF.BuildTools/tests/OperationParameterInjectionGeneratorTests.cs:1682]: Does leaving the attribute in the code but commented out have any purpose?

Comment (g7ed6e) [reply src/CoreWCF.BuildTools/tests/OperationParameterInjectionGeneratorTests.cs:1681]: Not really. It was there to make things more explicit. I just deleted.

**Comment by mconnew** [comment]
It's nice to see even more unit tests with this PR. I'll merge it once the tests have all passed.

**Pull request merged**